client/my-sites/people/people-invites/index.jsx

```diff
@@ -64,14 +66,19 @@ class PeopleInvites extends React.PureComponent {
 	};
 
 	render() {
-		const { site } = this.props;
+		const { site, isJetpack, isPrivate } = this.props;
 		const siteId = site && site.ID;
 
 		return (
 			<Main className="people-invites">
 				{ siteId && <QuerySiteInvites siteId={ siteId } /> }
 				<SidebarNavigation />
-				<PeopleSectionNav filter="invites" site={ site } />
+				<PeopleSectionNav
```

Comment:
We should also have `jetpackPeopleSupported` here for completeness (it really should be a blackbox to us as to what this prop controls). As done in

https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/people/main.jsx#L106